### DISCUSSION TITLE
In what instance is it useful to use "leave as is" on the restore initial scene setup dialog?

### DISCUSSION THREAD

**Discussion by melMass** [yasirkula/UnityAssetUsageDetector#31] [Q&A] [answered]
Hi,

First of thanks a lot, this tools is helping me clean an enormous project I need to properly clean/backup.
As it's a backup that I'm not touching anymore, I have ticked the `Faster Refresh` checkbox, and after the first run it's going relatively fast (depending on the assets usage of course), but I was wondering if use "leave as is" was making things even faster?

![image](https://user-images.githubusercontent.com/7041726/170844384-87a2ea24-945c-41b5-9822-c61d1a2d7ba4.png)

**Comment by yasirkula** [comment, marked as answer]
Hi,

This message says that you've started the search in NightClub scene but then switched over to another scene before initiating another search. If you say Yes, currently open scene will be closed and NightClub scene will be reopened. If you say Leave it as is, currently open scene won't be touched and NightClub scene won't be opened automatically. This won't affect search performance.

**Reply by melMass** [reply]
Thanks, the thing is that I'm just always using the nightclub scene but I was mainly wondering about perf so this answers my question
